xknx/devices/cover.py

```diff
@@ -271,14 +284,21 @@ async def set_angle(self, angle: int) -> None:
         if not self.supports_angle:
             logger.warning("Angle not supported for device %s", self.get_name())
             return
+
+        self.travel_direction_tilt = (
+            TravelStatus.DIRECTION_DOWN
+            if self.current_angle() and angle > int(self.angle.value)
```

Comment:
```suggestion
            _current_angle = self.current_angle()
            if _current_angle is not None and angle > _current_angle:
```
Check for `None` explicitly. Otherwise 
`angle: 100 - current_angle: 50 -> True` whereas 
`angle: 100 - current_angle: 0 -> False`
The suggestion should also satisfy mypy without using self.angle.value which is of type `Any`